src/components/DropdownMenu.tsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import styled from 'styled-components';
+import { ButtonVariant, applyButtonVariantStyles } from '../theme/buttons';
+import { palette } from '../theme/palette';
+
+const StyledDropdownMenu = styled.div`
+  position: relative;
+`;
+
+const StyledDropdownMenuButton = styled.button<{ variant: ButtonVariant; width?: string }>`
+  ${(props) => applyButtonVariantStyles(props.variant)}
+
+  align-items: center;
+  border: 0;
+  border-radius: 0.5rem;
+  box-shadow: 0px 0.2rem 0.4rem rgba(0, 0, 0, 0.2);
+  display: inline-flex;
+  flex-direction: row;
+  font-size: 1.6rem;
+  height: 2.5rem;
+  justify-content: center;
+  line-height: 2rem;
+  padding: 0 0.5rem;
+  text-decoration: none;
+  transition: all 0.2s ease-in-out;
+  user-select: none;
+  white-space: nowrap;
+  ${(props) => props.width ? `width: ${props.width}` : null}
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+
+  &:disabled {
+    opacity: 0.4;
+  }
+
+  &:not([disabled]) {
+    cursor: pointer;
+  }
+
+  :after {
+    background: ${(props) => props.variant === 'light' ? palette.black : palette.white};
+    clip-path: polygon(0 0, 100% 100%, 100% 0);
+    content: ' ';
+    display: block;
+    height: 0.5rem;
+    margin: -0.25rem 0 0 0.5rem;
+    transform: rotate(135deg);
+    width: 0.5rem;
+  }
+`;
+
+const StyledDropdownMenuItemContainer = styled.div`
+  background-color: ${palette.white};
+  border: 1px solid ${palette.pale};
+  padding: 0.3rem 0;
+  position: absolute;
+  z-index: 10;
+`;
+
+const StyledDropdownMenuItem = styled.button`
```

Comment:
at least call this `StyledDropdownMenuButton` or `StyledDropdownMenuItemButton` so that we could add links to it later

Reply:
Changed this to StyledDropdownMenuItemButton and the component that uses it to DropdownMenuItemButton